regex_macros/src/lib.rs

```diff
@@ -20,6 +20,7 @@
 extern crate regex;
 extern crate syntax;
 extern crate rustc;
```

Comment:
This line can actually be removed since rustc is only required for the plugin registry.